universal-login-commons/lib/core/models/relayer.ts

```diff
@@ -19,4 +19,5 @@ export interface PublicRelayerConfig {
   factoryAddress: string;
   chainSpec: ChainSpec;
   contractWhiteList: ContractWhiteList;
+  relayerAddress: string;
```

Comment:
Let's Use tx.origin instead of encoding address in SDK.

Reply:
✅